image/templates/helm/shared/templates/02-scanner-v4-07-db-deployment.yaml

```diff
@@ -102,7 +102,7 @@ spec:
           value: "/var/lib/postgresql/data/pgdata"
         - name: POSTGRES_HOST_AUTH_METHOD
           value: "scram-sha-256"
-        {{- include "srox.envVars" (list . "deployment" "scanner-v4-db" "scanner-v4-db") | nindent 8 }}
+        {{- include "srox.envVars" (list . "deployment" "scanner-v4-db" "db") | nindent 8 }}
```

Comment:
is the last thing the container name in the config? Do we need this for the init-db?

Reply:
Correct, it's the container name. Regarding your second question, I would say this is a matter of convention and documentation, but so far it seems we haven't included `srox.envVars`-style customization capabilities into our init containers. Hence I wouldn't change this at this moment.